int32 exit_code
int32 CODE_PEDICLE_DETECTION_SUCCESS = 1
int32 CODE_PEDICLE_DETECTION_FAILED = -210
